fitsmap/cartographer.py

```diff
@@ -269,9 +275,9 @@ def markers_to_js(marker_file_names: List[str]) -> str:
         "        for (j = 0; j < collection.length; j++){",
         "            src = collection[j];",
         "",
-        "            var width = (((src.widest_col * 10) * src.n_cols) + 10).toString() + 'em';",
+        "            var width = '900px';",
```

Comment:
Did you find that the width and height that were set before were too/small or not the right ratio? I would like to avoid hard-coding if possible.

Reply:
I didn't mean to commit this with the other changes.  I'm working on a change to revert it to your default and add a global variable to allow it to be modified.